core-plugins/src/main/java/co/cask/hydrator/plugin/batch/aggregator/RowDenormalizerConfig.java

```diff
@@ -112,12 +118,14 @@ String getFieldValue() {
    *
    * @return Map of output fields and its alias
    */
-  Map<String, String> getOutputFieldMappings() {
+  Map<String, String> getFieldAliases() {
     Map<String, String> outputFieldMappings = new HashMap<String, String>();
-    for (String field : Splitter.on(',').trimResults().split(outputFields)) {
-      String[] value = field.split(":");
-      if (value.length == 2) {
-        outputFieldMappings.put(value[0], value[1]);
+    if (fieldAliases != null) {
+      for (String field : Splitter.on(',').trimResults().split(fieldAliases)) {
+        String[] value = field.split(":");
+        if (value.length == 2) {
+          outputFieldMappings.put(value[0], value[1]);
+        }
```

Comment:
We are not throwing any exception because 'fieldAliases' is optional. So we are building map only when we will get the values from user, otherwise we are not doing anything.